### PR TITLE
fix(challenge): Improve example for "Applied Visual Design: Use the u Tag to Underline Text"

### LINKED ISSUE #12986: [beta] Improve example for "Applied Visual Design: Use the u Tag to Underline Text"
Challenge [use-the-u-tag-to-underline-text](http://beta.freecodecamp.com/en/challenges/applied-visual-design/use-the-u-tag-to-underline-text) has an issue.
User Agent is: <code>Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.76 Safari/537.36</code>.

```html
<style>
h4 {
  text-align: center;
  height: 25px;
}
p {
  text-align: justify;
}
.links {
  text-align: left;
  color: black;
}
.fullCard {
  width: 245px;
  border: 1px solid #ccc;
  border-radius: 5px;
  margin: 10px 5px;
  padding: 4px;
}
.cardContent {
  padding: 10px;
}
.cardText {
  margin-bottom: 30px;
}
</style>
<div class="fullCard">
<div class="cardContent">
  <div class="cardText">
    <strong><h4>Google</h4></strong>
    <p>Google was founded by Larry Page and Sergey Brin while they were Ph.D. students at Stanford University.</p>
  </div>
  <div class="cardLinks">
    <u><a href="https://en.wikipedia.org/wiki/Larry_Page" class="links">Larry Page</a><br><br>
      <a href="https://en.wikipedia.org/wiki/Sergey_Brin" class="links">Sergey Brin</a></u>
  </div>
</div>
</div>
```

Much like #12831, I think the example used in this challenge could be improved. Since the `<a>` tags already have a underline, there's no clear change from adding the `<u>` tags.

What do you think about changing this challenge to ask campers to wrap the words "Ph.D. students" like `<u>Ph.D. students</u>`?

This text formatting makes no sense practically - **but it makes a better example.**

### To make this change:
* Update the instructions.
* Update the tests.
* Update the `challengeSeed` for the following challenges.

This PR #12860 demonstrates what this update could look like.

### PR DESCRIPTION
**Pull request by Greenheart**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change
- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:
- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12986 

#### Description
This improves the example of the challenge according to the referenced issue.
* Update instruction.
* Update `challengeSeed` for affected challenges.
* Update tests.
* Use strict equality for tests.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 1]
```diff
@@ -451,9 +451,9 @@
         "</div>"
       ],
       "tests": [
-        "assert($('u').length == 1, 'message: Your code should add a <code>u</code> tag to the markup.');",
-        "assert($('u').children('a').length == 2, 'message: The <code>u</code> tag should wrap around and contain the two <code>a</code> tags.');",
-        "assert($('u').children('div').length == 0, 'message: The <code>u</code> tag should not wrap around the parent <code>div</code> tag.');"
+        "assert($('u').length === 1, 'message: Your code should add a <code>u</code> tag to the markup.');",
+        "assert($('u').text().indexOf('Ph.D. students') > -1, 'message: The <code>u</code> tag should wrap around the text \"Ph.D students\".');",
```
Comment (erictleung) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]:
```diff
- ...around the text \"Ph.D students\"
+ ...around the text \"Ph.D. students\"
```

**Review by erictleung — CHANGES_REQUESTED** [review]
Just did a static code review and found that one *small* inconsistency. I haven't seeded this locally yet to just double check. I'll try to get around it tomorrow, unless someone else gets to it before I do. But it looks all right so far.

File: seed/challenges/01-responsive-web-design/applied-visual-design.json (hunk 1 above)
Comment (Greenheart) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: @erictleung Thanks for sharing that formatted diff! I didn't know that :smile:

Comment (erictleung) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: @Greenheart no prob! Yeah, it blew my mind when I found out you can do that. The "language" for the code block is "diff" and you just have to add the `+` and `-` signs accordingly.

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 2]
```diff
@@ -407,7 +407,7 @@
       "description": [
         "To underline text, you can use the <code>u</code> tag. This is often used to signify that a section of text is important, or something to remember. With the <code>u</code> tag, the browser applies the CSS of <code>text-decoration: underline;</code> to the element.",
         "<hr>",
-        "Wrap the <code>u</code> tag around the two anchor tags. It should not include the parent <code>div</code> that has the class of <code>cardLinks</code>.",
+        "Wrap the <code>u</code> tag around the text \"Ph.D. students\". It should not include the parent <code>div</code> that has the class of <code>cardLinks</code>.",
```
Comment (raisedadead) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]:
```diff
-cardLinks
+cardText
```

Caught a pre-existing typo, @Greenheart would you mind fixing this?

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: seed/challenges/01-responsive-web-design/applied-visual-design.json (hunk 2 above)
Comment (Greenheart) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: @raisedadead Done! :smile:

**Review by raisedadead — APPROVED** [review]

**Review by erictleung — APPROVED** [review]

**Pull request merged**